src/customers/account-dashboard-my-purchase-orders.md

```diff
@@ -26,6 +26,17 @@ Purchase Orders create a _snapshot_ of item prices, discounts, and shipping pric
 {:.bs-callout-warning}
 If an item in a purchase order is out-of-stock, or has insufficient quantity available, when the purchase order is converted into an actual order, an error will occur. If backorders are enabled, the order will be processed normally.
 
+## Bulk approval/rejection
+
+As of Magneto 2.4.1, approvers may approve or reject mutliple purchase orders at once.
+
+1. On the _My Purchase Order_ page, go to the Requires My Approval tab.
+1. Using the checkboxes, select the purchase orders you wish to approve or reject.
+1. Click the <span class="btn">Approve Selected</span> or the <span class="btn">Reject Selected</span> button.
+
+You can only select purchase orders on which you can take an action. Others are grayed out.
```

Comment:
This should probably be nested in the selection step. "Grayed out" is discouraged because it is dev jargon. "Dimmed" can be used when necessary, but it is understood to indicate that an item is unavailable. 

```suggestion
You can select only the purchase orders with a current status that allows you can take an action.
```